shared/TypeDefinitions.ts

```diff
@@ -6,7 +6,9 @@ export type RootStackParamList = {
   StartScreen: undefined,
   TestModalScreen: undefined,
   FullScreen: undefined,
-  FormScreen: undefined,
+  FormScreen: {
+    isNewObservation: boolean,
```

Comment:
For tydelighets skyld kalte jeg denne isNewObservation istedenfor new da jeg merget fra master

Reply:
Ok :)

Reply:
Fixed